var/spack/repos/builtin/packages/h5bench/package.py

```diff
@@ -58,3 +59,33 @@ def cmake_args(self):
         ]
 
         return args
+
+    @run_after("install")
+    def setup_build_tests(self):
+        launcher = self.mpi_launcher()
+
+        filter_file(r"-n 2", "-n 2 --timeout 240", "samples/sync-write-1d-contig-contig.json")
+
+        """Copy the example source files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        self.cache_extra_test_sources(["tests", "samples"])
```

Comment:
I forgot this is also deprecated.  Use the following instead:

```suggestion
        cache_extra_test_sources(self, ["tests", "samples"])
```